c++/src/Reader.hh

```diff
@@ -254,7 +255,6 @@ namespace orc {
                                std::vector<std::vector<proto::ColumnStatistics> >* indexStats) const;
 
     // metadata
-    mutable std::unique_ptr<proto::Metadata> metadata;
     mutable bool isMetadataLoaded;
```

Comment:
Why is this flag not moved together with _**std::unique_ptr<proto::Metadata> metadata**_ ?

Reply:
Because the Metadata variable need to be consumed in RowReaderImpl class and the isMetadataLoaded flag don't be consumed in RowReaderImpl.